front/app/components/FollowUnfollow/index.tsx

```diff
@@ -135,21 +139,73 @@ const FollowUnfollow = ({
     loginAndFollow();
   };
 
+  const getTooltipContent = () => {
+    if (toolTipType === 'input') {
+      return (
+        <FormattedMessage
+          {...messages.followTooltipInputPage}
+          values={{
+            unsubscribeLink: (
+              <a
+                href={'/profile/edit'}
+                target="_blank"
+                rel="noreferrer"
+                style={{ textDecoration: 'underline', color: colors.white }}
+              >
+                <FormattedMessage {...messages.unsubscribe} />
+              </a>
+            ),
+          }}
+        />
+      );
+    } else if (toolTipType === 'projectOrFolder') {
+      return (
+        <FormattedMessage
+          {...messages.followTooltipProjects}
+          values={{
+            unsubscribeLink: (
+              <a
+                href={'/profile/edit'}
+                target="_blank"
+                rel="noreferrer"
+                style={{ textDecoration: 'underline', color: colors.white }}
+              >
+                <FormattedMessage {...messages.unsubscribe} />
+              </a>
+            ),
+          }}
+        />
+      );
+    }
+    return null;
+  };
+
   return (
-    <Button
-      buttonStyle={buttonStyle}
-      icon="notification"
-      onClick={handleButtonClick}
-      iconSize={iconSize}
-      px="12px"
-      processing={isLoading}
-      {...otherButtonProps}
-      data-cy={isFollowing ? 'e2e-unfollow-button' : 'e2e-follow-button'}
+    <Tippy
+      interactive={true}
+      disabled={false}
```

Comment:
```suggestion
      disabled={!tooltipType}
```

Right now, if you don't provide the `tooltipType`, I think it still shows an empty tooltip. If we don't have a `tooltipType` we should just disable the tooltip instead

Reply:
Good catch! Thanks Luuc!